### ISSUE TITLE
Recipent decryption

### ISSUE THREAD

**Issue by str4d** [str4d/wage#2]
This should take a decryptor and a list of identity files, and return the same value as `decrypt_with_passphrase`:
```rust
#[wasm_bindgen]
pub async decrypt_with_identities(decryptor: u32, identities: &[web_sys::File]) -> Result<JsValue, JsValue> {}
```

**Comment by fubuloubu** [comment]
I started looking at this, does `age::keys::Identity` have the right trait(s) to read from `web_sys::File`?

**Comment by str4d** [comment]
`File.stream()` returns `ReadableStream`, which we can wrap with `StreamReader` to get something implementing `AsyncRead` (as with the file being decrypted), but not the blocking `BufRead`. I think what we eventually want is an equivalent `Identity::from_async_buffer` in `age`, but we could also manually read each identity into a `secrecy::SecretVec` and then pass those through `Identity::from_buffer`.

**Comment by pirate** [comment]
Not sure if this issue is the right place for it, but if you need a snippet for this TODO in the readme `Expose a Rust stream to JavaScript as a user-downloadable file.`, this one works well:

```javascript
function download(content, filename, contentType) {
    contentType = contentType || 'application/octet-stream'
    const anchor = document.createElement('a')
    const blob = new Blob([content], {'type': contentType})
    anchor.href = window.URL.createObjectURL(blob)
    anchor.download = filename
    anchor.click()
}
```

**Comment by str4d** [comment]
Implemented in #25.
